xmps/specphot.pro.txt

```diff
@@ -0,0 +1,223 @@
+function specphot, spec1d, hdr,rlambda,rresponse,ilambda,iresponse
```

Comment:
Is there a reason to carry this IDL code around?
If not remove it.

Either way, find a better spot for it than xmps/